evennia/contrib/rpg/rpsystem/rpsystem.py

```diff
@@ -1539,27 +1397,26 @@ def get_display_name(self, looker, **kwargs):
 
         Keyword Args:
             pose (bool): Include the pose (if available) in the return.
+            ref (str):   Specifies the capitalization for the displayed name.
```

Comment:
Why `ref` as a name for this? Make the arg more readable and relevant. This also seems like it could do with an example for how it should look.